app/src/main/java/io/homeassistant/companion/android/sensors/SensorReceiver.kt

```diff
@@ -61,6 +62,27 @@ class SensorReceiver : BroadcastReceiver() {
             .build()
             .inject(this)
 
+        when (intent.action) {
+            "android.app.action.NEXT_ALARM_CLOCK_CHANGED" -> {
+                val sensorDao = AppDatabase.getInstance(context).sensorDao()
+                val sensor = sensorDao.get(NextAlarmManager.nextAlarm.id)
+                if (sensor?.enabled != true) {
+                    Log.d(TAG, "Alarm Sensor disabled, skipping sensors update")
+                    return
+                }
+            }
+            "android.bluetooth.device.action.ACL_CONNECTED",
+                "android.bluetooth.device.action.ACL_DISCONNECTED",
+                BluetoothAdapter.ACTION_STATE_CHANGED -> {
```

Comment:
So now we are adding a registered receiver too? I was thinking here we would just skip the ones we can't get rid of and find a better way to skip the register/unregister as needed, otherwise we would need to take care of all the other receivers on this page and it may get a bit out of hand :)

Reply:
Just for consistency - to filter out all bluetooth events. It might be removed from here when a better way to skip the register/unregister as needed will be found.